.github/workflows/UnitTests-core.yml

```diff
@@ -39,6 +41,8 @@ jobs:
           git fetch upstream
       - name: Setup Gradle
         uses: gradle/actions/setup-gradle@v4
+        with:
```

Comment:
What does this mean?

Reply:
By default gradle adds summary to the `Summary` page. We don't really use this info so I disabled it to avoid clutter on `Summary` page.